eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -333,11 +333,7 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode)
 Cache purge for content/locations is triggered when needed (e.g. on publish) and will result in one or several Http PURGE requests.
 Can be "local" or "http" or a valid service ID:
 - If "local" is used, an Http PURGE request will be emulated when needed (e.g. when using Symfony internal reverse proxy).
-- If "http" is used, only one Http BAN request will be sent, with X-Location-Id header containing locationIds to ban.
-  X-Location-Id consists in a Regexp containing locationIds to ban.
-  Examples:
-   - (123|456|789) => Purge locations #123, #456, #789.
-   - .* => Purge all locations.
+- If "http" is used, only one Http PURGE request will be sent, with xkey header containing tags to purge seperated by space.
```

Comment:
The updated VCL should handle both, as well as SymfonyCache. So yes.
But the MVC layer will only send `xkey` once this is merged, not `X-Location-Id`. Do you think that this is a problem ?